packages/app/components/token-picker/TokenPicker.tsx

```diff
@@ -1,69 +1,112 @@
 "use client";
 
+import { useTokenListContext } from "@/context/TokenListContext";
+import { Token } from "@/models/token/types";
 import {
   BodyText,
+  Icon,
   Modal,
   ModalBaseProps,
   ModalContent,
   ModalHeaderTitle,
+  RadioButton
 } from "@/ui";
+import { TokenIcon } from "../TokenIcon";
+import { TOKEN_PICKER_COMMON_TOKENS } from "./constants";
+
+interface TokenListRowProps {
+  token: Token;
+  closeAction: () => void;
+}
+
+interface TokenListProps {
+  closeAction: () => void;
+  tokenList: Token[];
+}
 
 export const TokenPicker = ({ isOpen, closeAction }: ModalBaseProps) => {
+  /**
+   * @TODO in STK-93: ensure we don't get duplicated token addresses
+   * @see https://linear.app/swaprdev/issue/STK-93/tokenlist-has-tokens-from-unsupported-chains
+   */
```

Comment:
Not a fan of leaving comments like this.  We already have the task to track this.

Reply:
I usually leave these comments as signaling which files should we touch for a task (if I know beforehand), I can remove them completely, or remove the `@see` tag. 

I tend to do this because when I (or someone) gets to the task, may be months from now and we lose context. I don't have strong opinions on leaving them or removing them, it's more about force of habit.